docs/book/how-to/customize-docker-builds/use-a-prebuilt-image.md

```diff
@@ -0,0 +1,114 @@
+# Skip building an image on every pipeline run
+
+When running a pipeline on a remote Stack, ZenML builds a Docker image with a base ZenML image and adds all of your project dependencies and your pipeline code to it. This process might take significant time depending on how big your dependencies are, how powerful your local system is and how fast your internet connection is (to pull base layers and push the final image to your container registry). 
+
+To save time and costs, you can choose to not build a Docker image every time your pipeline runs. This guide shows you how to do it using a pre-built image, what you should include in your image for the pipeline to run successfully and other tips.
+
+{% hint style="info" %}
+Note that using this feature means that you won't be able to leverage any updates you make to your code or dependencies, outside of what your image already contains.
+{% endhint %}
+
+The DockerSettings class in ZenML allows you to set a parent image to be used in your pipeline runs and the ability to skip building an image on top of it. This approach works in two ways:
+- [If ZenML built an image for your pipeline previously](#if-zenml-built-an-image-for-your-pipeline-previously)
+- [If ZenML has never built an image for your pipeline](#if-zenml-has-never-built-an-image-for-your-pipeline)
+
+## If ZenML built an image for your pipeline previously
```

Comment:
Does it matter if zenml has built the image before or not?

Reply:
It matters in the sense that if zenml had already built an image, you can just reuse it directly in your dockersettings with skip_build set to True. No worrying about adding the code and dependencies yourself

Reply:
@wjayesh I think this distinction is just weird to separate so clearly at this level. if anything maybe an addendum at the end of a section that you can reuse an image built by zenml before (you start the chapter with saying you want to avoid zenml building an image in the first place after all)

I think anytime users set skip_build=True they should really know whats happening

Reply:
i agree, it doesn't make sense to talk about that case when the title is about not using an image. i have amended it